tsconfig.json

```diff
@@ -1,15 +1,16 @@
 {
-    "compilerOptions": {
+	"compilerOptions": {
+		"allowSyntheticDefaultImports": true,
 		"allowJs": true,
+		"declaration": true,
+		"esModuleInterop": true,
 		"jsx": "react", // use typescript to transpile jsx to js
-		"lib": [ "dom", "es6" ],
+		"importHelpers": true,
+		"lib": ["dom", "es6"],
 		"moduleResolution": "node",
 		"outDir": "./assets/dist",
-		"strictNullChecks": true,
-        "target": "es6"
-    },
-    "include": [
-        "./assets/src/**/*"
-	],
+		"target": "es6"
+	},
+	"include": ["./assets/src/**/*"],
```

Comment:
Had some errors happening: 
![image](https://user-images.githubusercontent.com/1477453/70230695-8fcc0300-1761-11ea-8012-e622a6ce1a5f.png)
so I took a look at Blueprint's tsconfig files and updated our.